.github/workflows/test.yml

```diff
@@ -61,6 +61,17 @@ jobs:
         ln -sf $(pwd) src/github.com/ovn-org/ovn-kubernetes
         GOPATH=$(pwd) gcov2lcov -infile gover.coverprofile -outfile coverage.lcov
 
+    - name: Build docker image
+      run: |
+        echo ${{ secrets.GITHUB_TOKEN }} | docker login docker.pkg.github.com -u $GITHUB_ACTOR --password-stdin
+        pushd dist/images
+          sudo cp -f ../../go-controller/_output/go/bin/ovn* .
+          echo "ref: $(git rev-parse  --symbolic-full-name HEAD)  commit: $(git rev-parse  HEAD)" > git_info
+          docker build -t docker.pkg.github.com/${GITHUB_REPOSITORY}/ovn-daemonset-f:dev -f Dockerfile.fedora .
+          docker push docker.pkg.github.com/${GITHUB_REPOSITORY}/ovn-daemonset-f:dev
```

Comment:
```
3b53a1a1ef4a: Waiting
denied: Resource not accessible by integration
```
docker push seems only working in my fork :thinking:

Reply:
I found the problem, you can not use the github token from a fork PR
https://github.community/t/github-actions-are-severely-limited-on-prs/18179/2